### ISSUE TITLE
Doesn't work with multiline expressions

### ISSUE THREAD

**Issue by attilahorvath** [shanehofstetter/rails-i18n-vscode#20]
If I break my `I18n.t(...)` calls into separate lines to improve readability, the extension fails to pick up the keys and no tooltip is displayed on hover.

For example:

```ruby
I18n.t(
  'main_page.welcome_user',
  user_name: current_user.name,
  last_sign_in: current_user.last_sign_in_date
)
```

I tried to look into this and it seems the root of the issue is that this extension is using the `TextDocument.getWordRangeAtPosition()` API which only works for a single line:

> Note 2: A custom regular expression will fail to match multiline strings and in the name of speed regular expressions should not match words with spaces.

https://code.visualstudio.com/api/references/vscode-api#TextDocument

The docs suggest using `TextLine.text` as an alternative so that might work but I'm afraid I don't really know enough about it to investigate if it's possible to replace this call with that.

**Comment by shanehofstetter** [comment]
Hi @attilahorvath,
Yeah I'm afraid the current implementation of using a regex to detect the translate statement is a bit limited. I found a way to make it work with your example, by taking the previous line into account too, but this is rather a hack. It seems that VSCode does not really have an Api to help with this, I guess for stuff like this one has to use a language server/ast.

**Comment by attilahorvath** [comment]
Yeah, I had a feeling it wouldn't be too easy after going through the API docs myself. It would probably be a big effort to migrate to some other solution.

Thanks for looking into this, it's not a huge problem so I don't mind if you want to close this issue. The extension is still very useful! :+1:
